platform/ui/src/components/SegmentationGroupTable/SegmentationDropDownRow.tsx

```diff
@@ -122,8 +122,24 @@ function SegmentationDropDownRow({
         />
       )}
       <div className="flex items-center">
+        <Tooltip
+          position="left"
+          content={
+            <div className="flex w-[140px] flex-col">
+              <div className="text-[13px] text-white">Series:</div>
+              <div className="text-aqua-pale text-[13px]">{activeSegmentation.description}</div>
+            </div>
+          }
+        >
+          <Icon
+            name="info-action"
+            style={{
+              color: '#348CFD',
```

Comment:
please don't hard code the color, if we need a new color add it to tailwind config